docs/diginstroom/sip/2.0/profiles/material-artwork.md

```diff
@@ -0,0 +1,185 @@
+---
+layout:       default
+title:        Material artwork
+parent:       Profiles
+grand_parent:  2.0
+nav_order:    3
+nav_exclude:  false
+---
+Editor's Draft
+{: .label .label-yellow }
+# Profile: Material artwork 
+
+The _Material artwork_ profile supports digital reproductions of artworks that are moveable material objects, often, but not always, displayed in or maintained by museums. 
+This includes photographic registration of 2D artworks, such as paintings or drawings, in high and very high (gigapixel) resolution and high-polygon scans of 3D artworks, such as statues or sculptures.
+
+This content profile specifies how to package different media files (e.g., TIFF, JPEG, OBJ or MTL), their metadata and the relationships between them in a meemoo SIP package.
+It also allows extensions to the descriptive metadata using [Schema.org](https://schema.org).
+
+**Permalink:** <https://data.hetarchief.be/id/sip/2.0/material-artwork>
```

Comment:
All permalinks (or basically all links with https://data.hetarchief.be/id/) now resolve to unexisting, and closed of resources in the meemoo triplyDB PRD instance.